### DISCUSSION TITLE
Is it possible to have a tooltip on the map and defining large circle depending on the value on a particular city location?

### DISCUSSION THREAD

**Discussion by hyfydistro** [airbnb/visx#1313] [Q&A] [answered]
Is it possible to have a tooltip on the map and defining large circle depending on the value on a particular city location? Frankly, I'm new to the map chart or its possibilities. I was wondering if I could get something like this set up? how?
![tooltip-map-01](https://user-images.githubusercontent.com/24542308/129469424-0426887f-263d-4e34-a1f7-ef4049874923.png)

**Comment by williaster** [comment, marked as answer]
Hi @hyfydistro it seems like in this case you would trigger the tooltip when hovering a circle? In that case you just need to attach an `onPointerMove` or equivalent handler to 
- show the tooltip
- set the tooltip data to whatever the circle's data is

There are several examples in the gallery for triggering tooltips with content specific to the point that triggered the tooltip.

**Reply by hyfydistro** [reply]
Thanks! :D
